composer.json

```diff
@@ -64,7 +64,8 @@
         "ext-curl": "*",
         "mikey179/vfsstream": "~1.6",
         "simplesamlphp/simplesamlphp-module-adfs": "dev-master",
-        "simplesamlphp/simplesamlphp-test-framework": "^0.2.7"
+        "simplesamlphp/simplesamlphp-test-framework": "^1.0.5",
+        "simplesamlphp/xml-security": "^0.0.10"
```

Comment:
Some test XMLs have been moved from the framework to either the saml2 lib or the new xml-security library.